src/bp/core/config/config-writer.ts

```diff
@@ -74,29 +90,42 @@ export class BotConfigWriter {
     }
   }
 
-  private async _writeToFile(config: BotConfig) {
-    const filePath = this.getBotFilePath(config.id)
+  private async _writeConfig(botId: string, config: BotConfig) {
     const fileName = 'bot.config.json'
-
-    try {
-      await fse.ensureDir(filePath)
-      await fse.writeFile(filePath + fileName, JSON.stringify(config, undefined, 2))
-    } catch (e) {
-      throw new VError(e, `Error writing file "${filePath}"`)
-    }
+    const scopedGhost = this.ghost.forBot(botId)
+    await scopedGhost.upsertFile('/', fileName, JSON.stringify(config, undefined, 2))
   }
 
   async deleteBotFolder(botId: string) {
-    const filePath = this.getBotFilePath(botId)
+    const scopedGhost = this.ghost.forBot(botId)
+    await scopedGhost.deleteFile('/', 'bot.config.json')
```

Comment:
do you only want to delete de the bot config file or all of the bot content ? since this is BotConfigWriter it might not be it's job but where do you actually delete the bot data, models, actions, etc ?